.ci/run-link-check-plugin.sh

```diff
@@ -9,12 +9,26 @@ mvn --version
 curl --fail-with-body -I https://sourceforge.net/projects/checkstyle/
 mvn -e --no-transfer-progress clean site -Dcheckstyle.ant.skip=true -DskipTests -DskipITs \
    -Dpmd.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true
+mkdir -p .ci-temp
+
 echo "------------ grep of linkcheck.html--BEGIN"
 # "grep ... | cat" is required command is running in "set -e" mode and
 # grep could return exit code 1 if nothing is matching
-grep externalLink target/site/linkcheck.html | cat
+grep -E "doesn't exist|externalLink" target/site/linkcheck.html | grep -v 'Read timed out' \
+  |  cat > .ci-temp/linkcheck-errors.txt
+
+sort config/linkcheck-suppressions.txt > .ci-temp/linkcheck-suppressions-sorted.txt
```

Comment:
Why not pipe to sort in original grep?

Reply:
The idea was readability.

Reply:
Looks fine with pipe in original grep too though. Done.